device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -1597,6 +1597,7 @@ private void addToCallbackQueue(IotHubTransportPacket packet)
 
     private DeviceClientConfig getDefaultConfig()
     {
+        //noinspection LoopStatementThatDoesntLoop
```

Comment:
The alternative is to write out the full ```this.deviceClientConfigs.values().iterator().hasNext()``` and so on, and the current implementation seems easier to read